src/js/experimental/media-chrome-listbox.js

```diff
@@ -234,19 +246,27 @@ class MediaChromeListbox extends window.HTMLElement {
     return composedPath[index];
   }
 
-  handleSelection(e) {
+  handleSelection(e, toggle) {
     const item = this.#getItem(e);
 
     if (!item) return;
 
-    const selected = item.getAttribute('aria-selected') === 'true';
+    this.#selectItem(item, toggle);
+  }
 
+  #selectItem(item, toggle) {
```

Comment:
allows to *always* select an item, or toggle its selected state. This means that via the mouse, it'll only toggle with ctrl/command, but will always toggle with space/enter. Probably still needs to be tweaked some more, but good enough for this PR.

Reply:
I've updated the keypress handling to only allow toggling with the keyboard when the listbox is multi-selectable.